study.md

```diff
@@ -95,11 +98,11 @@ would use to satisfy these requirements.
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+<!-- Seed data is a clean way to set up a databse for a new app -->
 ```
 
 Should you use seeds to create data to experiment with during development?
 
 ```md
-<!-- your response here -->
+<!-- Yes this would be a good method of debuging problems. -->
 ```
```

Comment:
No. Seed data is *only* for data *required* by your app to function. Experimental data is convenient, not necessary.